lib/src/allocation.dart

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:ffi';
+import 'dart:io';
+
+// Note that kernel32.dll is the correct name in both 32-bit and 64-bit.
+final DynamicLibrary stdlib = Platform.isWindows
+    ? DynamicLibrary.open("kernel32.dll")
+    : DynamicLibrary.process();
+
+typedef PosixMallocFn = Pointer Function(IntPtr);
```

Comment:
Both, `PosixMallocFn` and `PosixMalloc`, are function types. Consider using something with "native" or "n" to distinguish them, e.g. `PosixMallocNFT` and `PosixMallocFT`.

Reply:
Done.